app/graphql/association_loader.rb

```diff
@@ -4,18 +4,19 @@ def self.validate(model, association_name)
     nil
   end
 
-  def initialize(model, association_name)
+  def initialize(model, association_name, token: nil, policy: nil)
```

Comment:
I wonder if there is a more automatic way to get a token in here.

Reply:
Afraid not.  We have User.current but nothing for Token.  I might add that when we move to Genkan